svthalia/concrexit
# GSuite changes

### Summary

These changes will allow members to use the web inbox functionality of google groups if they want to.

### How to test
Steps to test the changes you made:
1. Try a GSuite sync
2. Check the groups.google.com UI for features
